x-pack/plugins/ml/server/plugin.ts

```diff
@@ -93,6 +99,11 @@ export class MlServerPlugin
   private isMlReady: Promise<void>;
   private setMlReady: () => void = () => {};
   private savedObjectsSyncService: SavedObjectsSyncService;
+  private enabledFeatures: MlFeatures = {
```

Comment:
would it be possible to disable/enable some features during runtime? perhaps this value should be an Observable in that case.

Reply:
It is possible to switch serverless project while kibana is running, when this happens the kibana server performs a hard restart so it can initialise a different serverless plugin. It is during this serverless plugin setup that our `setFeaturesEnabled` function is called setting these `enabledFeatures` flags.